.github/CODEOWNERS

```
@@ -11,7 +11,16 @@
 .github/CODEOWNERS @edolstra
 
 # Documentation of built-in functions
-src/libexpr/primops.cc @roberth
+src/libexpr/primops.cc @roberth @fricklerhandwerk
+
+# Documentation of settings
+src/libexpr/eval-settings.hh @fricklerhandwerk
+src/libstore/globals.hh @fricklerhandwerk
+
+# Documentation
+doc/manual @fricklerhandwerk
+maintainers/*.md @fricklerhandwerk
+src/**/*.md
```

what does this line do?

---

It's supposed to capture all `.md` files in any nested directory.

---

I thought this would be invalid syntax, but GitHub documents it as:

```
# In this example, @octocat owns any file in the `/apps`
# directory in the root of your repository except for the `/apps/github`
# subdirectory, as its owners are left empty.
/apps/ @octocat
/apps/github
```

So it appears to exclude a directory from explicit code-ownership?

(Though I'm not sure what the intention was either)

---

D'oh, not adding my handle was of course a typo.

```suggestion
src/**/*.md @fricklerhandwerk
```